### PR TITLE
Vgg19 bn

### LINKED ISSUE #1: Current model accuracy at only 68%
Confusion matrix shows misclassification for mainly Fear, Sad, Neutral classes. Need to clean dataset to improve model accuracy.

### PR DESCRIPTION
**Pull request by nonsocchi**
Changed pre-trained model to `vgg19_bn`, and resized images to `224x224` to match new arch.

### CODE REVIEW

**Comment by nonsocchi** [comment]
Switched arch to `vgg19_bn` and reseized images to `128x128`. accuracy increased to `72%` using same number of epochs as in the main branch.
